src/app/space/create/deployments/deployments-donut/deployments-donut-chart/deployments-donut-chart.component.html

```diff
@@ -0,0 +1,22 @@
+<div #chartElement id="{{chartId}}" class="deployments-donut-chart" [ngClass]="{'mini': mini}"></div>
+<div *ngIf="mini" class="deployments-donut-chart-mini-text">
+  <span *ngIf="!idled">
+    {{total}}
+    <span *ngIf="total === 1">pod</span>
+    <span *ngIf="total !== 1">pods</span>
+  </span>
+  <span *ngIf="idled">
+    Idle
+  </span>
+</div>
+
+<div class="sr-only">
+  <div *ngIf="pods?.length === 0">No pods.</div>
+  <div *ngIf="pods?.length !== 0">
+    Pod status:
+    <li *ngFor="let column of podStatusData">
+      <span *ngIf="column[1]">{{column[1]}} {{column[0]}}</span>
+      <span *ngIf="!column[1]">0 {{column[0]}}</span>"
+    </li>
+  </div>
```

Comment:
Same with the ngIf-else here. What type is "pods"? It's `any` in the component TS - an array?